File: PROGRAMMERS/lv0/순서쌍의 개수.js
function solution(n) {
  let ans = 0;
  for (let i = 1; i <= Math.sqrt(n); i++) {
    if (n % i == 0) {
      if (i == Math.sqrt(n)) {
        ans++;
        continue;
      }
      ans += 2;
    }
  }
  return ans;
}

let inps = [20, 100];

for (let i of inps) {
  console.log(solution(i));
}

File: PROGRAMMERS/lv0/python/순서쌍의 개수.py
def solution(n):
    s = set()
    
    for v in range(1, int(n**0.5)+1):
        if(n%v==0):
            s.add(v)
            s.add(n/v)
            
    return len(s)

inps = [
    20,
    100
]

for inp in inps:
    print(solution(inp))